projects/adf4368/README.rst

```diff
@@ -0,0 +1,428 @@
+    | Pin | Name              | Description                              |
+    +-----+-------------------+------------------------------------------+
+    | 1   | CSB               | Connected to CSB of the ADF4368          |
+    +-----+-------------------+------------------------------------------+
+    | 2   | NC                | Not connected                            |
+    +-----+-------------------+------------------------------------------+
+    | 3   | SDIO              | Connected to SDIO of the ADF4368         |
+    +-----+-------------------+------------------------------------------+
+    | 4   | NC                | Not connected                            |
+    +-----+-------------------+------------------------------------------+
+    | 5   | SDO               | Connected to SDO of the ADF4368          |
+    +-----+-------------------+------------------------------------------+
+    | 6   | NC                | Not connected                            |
+    +-----+-------------------+------------------------------------------+
+    | 7   | SCLK              | Connected to SCLK of the ADF4368         |
+    +-----+-------------------+------------------------------------------+
+    | 8   | CSB               | Connected to CSB of the ADF4368          |
+    +-----+-------------------+------------------------------------------+
+    | 9   | GND               | Connected to Ground                      |
+    +-----+-------------------+------------------------------------------+
+    | 10  | NC                | Not connected                            |
+    +-----+-------------------+------------------------------------------+
+
+No-OS Build Setup
+-----------------
+
+Please see: https://wiki.analog.com/resources/no-os/build
+
+No-OS Supported Examples
+------------------------
+
+The initialization data used in the examples is taken out from:
+`Project Common Data Path <https://github.com/analogdevicesinc/no-OS/tree/master/projects/adf4368/src/common>`_
+
+The macros used in Common Data are defined in platform specific files found in:
+`Project Platform Configuration Path <https://github.com/analogdevicesinc/no-OS/tree/master/projects/adf4368/src/platform>`_
+
+Basic example
+^^^^^^^^^^^^^
+
+This example initializes the ADF4368 with the configurations provided in the
+above mentioned common files and applies them to the IC. By default reference
+frequency is provided by the local oscillator of the board and both channels are
+active with a power level of 9. Subsequently the example sets a test frequency
+of 8 GHz and adjusts the phase by 4ps.
+
+
+In order to build the basic example make sure you have the following
+configuration in the Makefile
+`Makefile <https://github.com/analogdevicesinc/no-OS/tree/master/projects/adf4368/Makefile>`_
+
+.. code-block:: bash
+
+	# Select the example you want to enable by choosing y for enabling and n for disabling
+	BASIC_EXAMPLE = y
+	IIO_EXAMPLE = n
```

Comment:
this will need to change when you switch to the `make EXAMPLE=basic_example` approach